test/functional/ui/statusline_spec.lua

```diff
@@ -0,0 +1,317 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
the rest of `global_statusline_spec.lua` could be moved into here. Doesn't look like we need a separate test file for a particular statusline configuration.

Reply:
That's already been done